apps/judicial-system/web/src/components/SigningModal/SigningModal.tsx

```diff
@@ -179,7 +179,7 @@ const SigningModal: React.FC<SigningModalProps> = ({
         signingProgress === 'inProgress'
           ? ''
           : signingProgress === 'success'
-          ? 'Senda ábendingu'
+          ? ''
```

Comment:
Don't we need some text here? Will it not result in an empty button?

Reply:
No, it results in no button at all. Just what we want.